config/defaults/secure-open-banking/policy-evaluation-script.js

```diff
@@ -297,18 +298,37 @@ if (intentType === "accountAccessIntent") {
     }
 
 } else if (intentType === "domesticPaymentIntent") {
+    logger.message("OB_Policy Domestic Payments Intent");
 
     var status = intent.Data.Status
-    var permissions = intent.Data.Permissions
-    var account = intent.account
+    var userResourceOwner = new Array(intent.user._id)
 
-    if (status != STATUS_AUTHORISED) {
-        logger.warning("Rejecting request - status [" + status + "]")
+    if (status !== STATUS_AUTHORISED) {
+        logger.error("Rejecting request - status [" + status + "]")
         authorized = false
     } else {
-        authorized = initiationMatch(environment.get("initiation").iterator().next(), intent.Data.Initiation)
+        responseAttributes.put("userResourceOwner", userResourceOwner);
+        var requestMethod = environment.get("request_method").iterator().next()
+
+        if (requestMethod != null) {
+            switch (requestMethod) {
+                case "POST":
+                    logger.message("OB_Policy Domestic Payments: POST request");
+                    var initiation = environment.get("initiation").iterator().next()
+                    authorized = initiationMatch(initiation, intent.Data.Initiation)
+                    break;
+                case "GET":
+                    logger.message("OB_Policy Domestic Payments: GET request");
+                    authorized = true
+                    break;
+                default:
+                    authorized = false
+                    break;
+            }
+        } else {
+            authorized = false
+        }
     }
-
 } else {
     authorized = false
-}
+}
```

Comment:
Can we have a final debug level log message that logs if the request was authorized or not? It will be handy if we have issues and need to work out the results of policy evaluation live in a deployment.

Reply:
Nice PR, some improvements in the logs could be nice to check the results of the policy evaluation.

Reply:
I added a log message at the end of the script, saying if the request was authorized or not.